test/wasi/test-wasi-initialize-validation.js

```diff
@@ -78,7 +78,8 @@ const bufferSource = fixtures.readSync('simple.wasm');
       () => { wasi.initialize(instance); },
       {
         code: 'ERR_INVALID_ARG_TYPE',
-        message: /"instance\.exports\._start" property must be undefined/
+        message: new RegExp('"instance\.exports\._start" property must be' + 
+          ' undefined\. Received function _start')
```

Comment:
I don't think using a regex makes sense here, you should drop it.
```suggestion
        message: '"instance\.exports\._start" property must be' + 
          ' undefined\. Received function _start',
```